src/main/java/com/ifmo/epampractice/entity/InvoiceEntity.java

```diff
@@ -0,0 +1,80 @@
+package com.ifmo.epampractice.entity;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class InvoiceEntity {
+    private int id;
+    private int orderId;
+    private Date issueDate;
+    private Date paymentDate;
+    private int totalPrice;
+
+    public InvoiceEntity() {
+    }
+
+    public InvoiceEntity(int id, int orderId, Date issueDate, Date paymentDate, int totalPrice) {
+        this.id = id;
+        this.orderId = orderId;
+        this.issueDate = issueDate;
+        this.paymentDate = paymentDate;
+        this.totalPrice = totalPrice;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Where is braces {} ?
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf 12 page